images/scripts/fedora.setup

```diff
@@ -40,6 +40,7 @@ openssl \
 PackageKit \
 pcp \
 pcp-libs \
+python3-pip \
```

Comment:
Note: This is temporary. Eventually this needs to go into the mock environment, not the VM host.

Reply:
Maybe both?